src/SprykerMiddleware/Zed/Process/Communication/Plugin/Processor/IntrospectionProcessorPlugin.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
```

Comment:
Sometimes we have this header, sometimes not.

Reply:
I added header to all files